playbooks/fedora-31/setup-env.yaml

```diff
@@ -5,6 +5,7 @@
       become: yes
       package:
         name:
+          - golang
```

Comment:
It would be nicer to split this part out into a separate and smaller (eg., *add Go to the list of build requirements*) commit, instead of having it as part of a larger *update tests* commit.